src/io/darwin.zig

```diff
@@ -1017,4 +1017,19 @@ pub const IO = struct {
             else => |e| e,
         };
     }
+
+    pub fn write_now(_: *IO, fd: posix.fd_t, buffer: []const u8) posix.WriteError!void {
```

Comment:
Let's call this `read_blocking`. The `_now` convention is four functions that can return `WouldBlock / null`